.github/workflows/refresh-hla.yml

```diff
@@ -0,0 +1,42 @@
+name: Update HLA
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  refresh:
+    name: Refresh HLA sequences
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v2
+        - uses: actions/setup-python@v2
+          with:
+            python-version: '3.6'
+        - name: Refresh HLA sequences
+          id: check_files
+          run: |
+            make refresh-hla-seqs
+            FNAME=$(git diff --name-only ontology/chain-sequence.tsv)
+            if [[ $FNAME == ontology/chain-sequence.tsv ]]; then
+              echo "::set-output name=has_change::true"
+            else
+              echo "::set-output name=has_change::false"
+            fi
+        - name: Create commit
+          id: create_commit
+          if: steps.check_files.outputs.has_change == 'true'
+          run: |
+            git config user.name 'github-actions'
+            git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
```

Comment:
This is the email address for the GitHub actions bot, so that the commit comes from the bot: https://github.community/t/github-actions-bot-email-address/17204

Reply:
Weird but I guess that's legit. Thanks!